From 30ef3e1397c39422dae886662b47d984936a186c Mon Sep 17 00:00:00 2001
From: Haoran Wang <elven.wang@nxp.com>
Date: Mon, 16 Dec 2024 10:58:15 +0800
Subject: [PATCH] Add missing header file

Added missing sys/uio.h header file to fix writev() missing issue.

Upstream-Status: Inappropriate  [platform specific changes]

Signed-off-by: Haoran Wang <elven.wang@nxp.com>
---
 tayga.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tayga.h b/tayga.h
index 732bc3d..ed529b1 100644
--- a/tayga.h
+++ b/tayga.h
@@ -17,6 +17,7 @@
 
 #include <stdio.h>
 #include <sys/types.h>
+#include <sys/uio.h>
 #include <sys/stat.h>
 #include <sys/ioctl.h>
 #include <sys/socket.h>
-- 
2.34.1

